protostar/cheatable_starknet/callable_hint_locals/prepare_hint_local.py

```diff
@@ -22,37 +21,37 @@ def __init__(self, contracts_controller: ContractsController):
 
     @property
     def name(self) -> str:
-        return "prepare"
+        return "prepare_tp"
 
     def _build(self) -> Callable[[Any], Any]:
         return self.prepare
 
     def prepare(
         self,
-        declared: DeclaredContract,
-        constructor_calldata: Optional[CairoOrPythonData] = None,
-        salt: Optional[int] = None,
+        class_hash: int,
```

Comment:
Why did you rename this? I think we should be consistent, in all other places it is called `constructor_calldata`.

Reply:
Arguments must have these exact names due to the way we translate this cheatcode to casm in compiler (it is passed as **kwargs in protostar code).
https://github.com/software-mansion-labs/cairo/blob/main/crates/cairo-lang-casm/src/hints/mod.rs#L610